cirq-web/cirq_web/bloch_sphere/bloch_sphere_test.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+import json
+
+import pytest
+import numpy as np
+
+import cirq_web
+
+from cirq.qis import to_valid_state_vector
+from cirq.qis.states import bloch_vector_from_state_vector
+
+
+def get_bundle_file_path():
+    # Need to call this from the root directory
+    absolute_path = cirq_web.resolve_path()
+    bundle_file_path = f'{absolute_path}/cirq_ts/dist/bloch_sphere.bundle.js'
+    bundle_script = cirq_web.to_script_tag(bundle_file_path)
+    return bundle_script
+
+
+def test_init_bloch_sphere_type():
+    bloch_sphere = cirq_web.BlochSphere()
+    assert isinstance(bloch_sphere, cirq_web.BlochSphere)
+
+
+@pytest.mark.parametrize('sphere_radius', [5, 0.2, 100])
+def test_valid_bloch_sphere_radius_json_info(sphere_radius):
+    bloch_sphere = cirq_web.BlochSphere(sphere_radius=sphere_radius)
```

Comment:
we should hook up cirq json serialization instead of rolling our own, so that cirq.to_json() and cirq.from_json() works well with these items as well.